### PR TITLE
Add Support for Swagger in API mlnet-web-embedded Template

### PR DESCRIPTION
**Pull request by DaviRamos**
Hi @aslotte Alex I created an implementation for the use of Swagger in the API of mlnet-web-embedded Template.

Please evaluate if it is good for use in the project

### CODE REVIEW

**Comment by DaviRamos** [comment]
Solve the Issue_5

File: src/ML.NET.Templates/ML.NET.Templates.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (aslotte) [inline src/ML.NET.Templates/ML.NET.Templates.sln:2]: As this is a project template, we won't need this solution file

File: src/ML.NET.Templates/ML.NET.Templates.csproj [hunk 2]
```diff
@@ -21,4 +21,6 @@
     <Compile Remove="**\*" />
   </ItemGroup>
 
+  <ProjectExtensions><VisualStudio><UserProperties templates_4ml_1net_1web_1embedded_4_1template_1config_4template_1json__JsonSchema="https://jenkins-x.io/schemas/jx-requirements.json" /></VisualStudio></ProjectExtensions>
```
Comment (aslotte) [inline src/ML.NET.Templates/ML.NET.Templates.csproj:24]: Do you know what this entry accomplishes in the `csproj` file?

**Review by aslotte — COMMENTED** [review]
Thank @DaviRamos! I added a couple of comments, let me know your thoughts.

File: src/ML.NET.Templates/ML.NET.Templates.sln (hunk 1 above)
Comment (DaviRamos) [reply src/ML.NET.Templates/ML.NET.Templates.sln:2]: The strength of the Habit is right here.

File: src/ML.NET.Templates/ML.NET.Templates.csproj (hunk 2 above)
Comment (DaviRamos) [reply src/ML.NET.Templates/ML.NET.Templates.csproj:24]: I was unable to reproduce this behavior.

I believe it was something automatically generated by the Swagger extension.

I did several tests and the behavior was not repeated ...

As everything was functional without it I decided to just remove this line.

@aslotte Do you prefer to make a ninth commit or do you directly correct these pending issues

Comment (aslotte) [inline src/ML.NET.Templates/ML.NET.Templates.csproj:24]:
```suggestion
```

**Review by aslotte — APPROVED** [review]
@DaviRamos I removed the solution file and the extra line in the csproj file. The rest looks great, thank you for your contribution! 👍

**Pull request merged**